pkgs/tools/security/logkeys/default.nix

```diff
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation rec {
   name = "logkeys-${version}";
-  version = "5ef6b0dcb9e3";
+  version = "78321c6e70f6";
```

Comment:
Please change version.
`If package is fetched from git's commit then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format.`